_sass/_color-matrix.scss

```diff
@@ -0,0 +1,170 @@
+// "Marge" is a cross between USWDS's "Medium" and "Large" breakpoints.
+// It's the equivalent of Bootstrap's "Medium" size.
+$marge-screen: 992px;
+
+// The WAI-ARIA specification states the following about aria-hidden:
+//
+//   Authors MAY, with caution, use aria-hidden to hide visibly rendered
+//   content from assistive technologies only if the act of hiding this
+//   content is intended to improve the experience for users of assistive
+//   technologies by removing redundant or extraneous content. Authors
+//   using aria-hidden to hide visible content from screen readers MUST
+//   ensure that identical or equivalent meaning and functionality is
+//   exposed to assistive technologies.
+//
+// Unfortunately, however, the U.S. Web Design Standards forcibly set
+// anything with aria-hidden to display: none, making it difficult to
+// address the above use case. The following rule allows us to override
+// this default and create content that is visible to sighted users but
+// irrelevant to non-sighted users.
+.usa-sr-invisible[aria-hidden="true"] {
```

Comment:
I don't know a whole bunch about how the two work together, but I'm inclined to agree with you @toolness — if upgrading is likely to introduce a lot of other conflicts, let's keep this issue focused on the color matrix, and file another for upgrading to the latest web design standards package.